Gems/Atom/Feature/Common/Code/Source/ProfilingCaptureSystemComponent.h

```diff
@@ -84,6 +87,13 @@ namespace AZ
             DelayedQueryCaptureHelper m_pipelineStatisticsCapture;
             DelayedQueryCaptureHelper m_cpuProfilingStatisticsCapture;
             DelayedQueryCaptureHelper m_benchmarkMetadataCapture;
+
+            // Stores the last continuous capture data so that the IO thread can access it by reference. 
+            // Ensure that this is not cleared until m_cpuDataSerializationInProgress is false. 
+            AZStd::deque<RHI::CpuProfiler::TimeRegionMap> m_lastContinuousCapture;
```

Comment:
Note: I looked into the lifetime of `ProfilingCaptureSystemComponent` and it appears that the component is only destructed when the renderer is destructed (see `CommonModule.cpp`) so this deque should always be alive while the IO thread is working.